backend/tests/command.rs

```diff
@@ -0,0 +1,144 @@
+use mensa_app_backend::{
+    interface::api_command::{AuthInfo, Command, InnerAuthInfo},
+    layer::{
+        data::{
+            database::factory::DataAccessFactory, flickr_api::flickr_api_handler::FlickrApiHandler,
+            mail::mail_sender::MailSender,
+        },
+        logic::api_command::command_handler::CommandHandler,
+    },
+    startup::config::ConfigReader,
+    util::{ReportReason, Uuid},
+};
+
+#[tokio::test]
+#[ignore = "manual test"]
+async fn test_add_image() {
+    let cmd = setup_cmd().await;
+
+    let meal = Uuid::try_from("48b0ed7b-8387-46ad-866c-7993f469a9bf").unwrap();
```

Comment:
Vielleicht mit lazy_static! als Konstante initialisieren, da dieselbe UUID immer wieder verwendet wird. Dasselbe gilt für die Image id in anderen Testfällen